reference/scalability_guide/more_than_5M_product_values.rst

```diff
@@ -1,7 +1,7 @@
 More than 5 millions of product values?
 ---------------------------------------
 
-Depending on your data volume (number of products, number of attributes per product, number of basic, localized or scopable attributes, number of locales, number of scopes, etc.), you will have to choose between the way your products are stored.
-Please read the following guide:
+Depending on your data volume (number of products, number of attributes per product, number of basic #(?)#, localized or scopable attributes, number of locales, number of scopes, etc.), you will have to choose a strategy to store your products.
```

Comment:
The equivalent of our "Field" then ? Should we call it like this ? Maybe "Global attribute" if that's what people outside our team is calling it.